src/views/GameListView.jsx

```diff
@@ -302,7 +304,9 @@ const GameListView = () => {
             <li className="duration-filter">
               <span className="filter-title">{t("Duration (mins)")}</span>
               <SliderRange
-                defaultValue={[15, 90]}
+                defaultValue={[2, 4]}
```

Comment:
Bad default values?

Reply:
Seems like a bad copy/paste to me. Fixing.